### PR TITLE
DIV-5844 - Update ithc

### PR DESCRIPTION
**Pull request by hosainnet**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines

### JIRA link (if applicable) ###

DIV-5844

### Change description ###

Merge master into ITHC to allow ITHC testing for xui

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_parameterized [hunk 1]
```diff
@@ -10,6 +9,7 @@ properties([
 		])
 ])
 
-Subscription subscription = new Subscription(env)
 
-sharedInfrastructurePipeline('div', params.ENVIRONMENT, subscription.nonProdName)
+withInfraPipeline('div') {
+	enableSlackNotifications('#div-dev')
+}
```
Comment (hosainnet) [inline Jenkinsfile_parameterized]: Updates from master

File: app-gateway.tf [hunk 2]
```diff
@@ -24,7 +24,7 @@ locals {
 }
 
 module "appGw" {
-  source            = "git@github.com:hmcts/cnp-module-waf?ref=master"
+  source            = "git@github.com:hmcts/cnp-module-waf?ref=ccd/CHG0033576"
```
Comment (hosainnet) [inline app-gateway.tf]: Point WAF to detection mode brance due to false positives

File: asp.tf [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
+//  source              = "git@github.com:hmcts/cnp-module-app-service-plan?ref=master"
+//  location            = "${var.location}"
+//  env                 = "${var.env}"
+//  resource_group_name = "${azurerm_resource_group.rg.name}"
+//  asp_sku_size        = "${var.asp_sku_size}"
+//  asp_capacity        = "${var.asp_capacity}"
+//  asp_name            = "${var.product}"
+//  ase_name            = "${local.ase_name}"
+//  tag_list            = "${var.common_tags}"
+//}
```
Comment (hosainnet) [inline asp.tf]: Comment out ASP so that it doesn't build it in Azure as we're moving to AKS

Comment (qzhou-hmcts) [reply asp.tf]: Still needed  for FE apps?

Comment (hosainnet) [reply asp.tf]: As per https://github.com/hmcts/div-shared-infrastructure/pull/53#discussion_r329478669

**Pull request merged**